src/json-completion.ts

```diff
@@ -354,7 +392,7 @@ export class JSONCompletion {
         let snippetValue = JSON.stringify(value);
         snippetValue = snippetValue.substr(1, snippetValue.length - 2); // remove quotes
         snippetValue = this.getInsertTextForPlainText(snippetValue); // escape \ and }
-        return '"${' + snippetValue + '}"' + separatorAfter;
+        return '"${' + snippetValue + '}"';
```

Comment:
I think we should keep the `separatorAfter` here? Or remove it from all the other places as well. We would easily miss that this was removed whenever we decide to implement the logic for `separatorAfter`

Reply:
indeed, I will wait until we have tests and test cases for this functionality in particular, and rebase this PR, and remove some of these "hacks"